examples/lighting-app/silabs/include/ColorFormat.h

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
This should be placed in a common silabs folder. Available for other apps. But ideally I think it should be part of the RGB widget

Reply:
It would be good to add below all LED related files in a new folder called ` connectedhomeip/examples/platform/silabs/led/`
- connectedhomeip/examples/platform/silabs/LEDWidget.cpp
- connectedhomeip/examples/platform/silabs/LEDWidget.h
- connectedhomeip/examples/lighting-app/silabs/src/ColorFormat.cpp
- connectedhomeip/examples/lighting-app/silabs/include/ColorFormat.h

Please suggest.

Reply:
Added below files into` connectedhomeip/examples/platform/silabs/led/`
 connectedhomeip/examples/lighting-app/silabs/src/ColorFormat.cpp
connectedhomeip/examples/lighting-app/silabs/include/ColorFormat.h

Reply:
created separate RGBLEDWidget class and moved src and header in a folder `connectedhomeip/examples/platform/silabs/rgb_led`
connectedhomeip/examples/platform/silabs/rgb_led/RGBLEDWidget.cpp
connectedhomeip/examples/platform/silabs/rgb_led/RGBLEDWidget.h